modules/local/sentieon/datametrics.nf

```diff
@@ -0,0 +1,62 @@
+process SENTIEON_DATAMETRICS {
+    tag "$meta.id"
+    label 'process_high'
+    label 'sentieon'
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path fasta
+    path fai
+
+    output:
+    tuple val(meta), path('*mq_metrics.txt')   , emit: mq_metrics
+    tuple val(meta), path('*qd_metrics.txt')   , emit: qd_metrics
+    tuple val(meta), path('*gc_summary.txt')   , emit: gc_summary
+    tuple val(meta), path('*gc_metrics.txt')   , emit: gc_metrics
+    tuple val(meta), path('*aln_metrics.txt')  , emit: aln_metrics
+    tuple val(meta), path('*is_metrics.txt')   , emit: is_metrics
+    path  "versions.yml"                       , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args ?: ''
+    def input  = bam ? '-i ' + bam.sort().join(' -i ') : ''
+    def ref    = fasta ? "-r $fasta" : ''
+    def prefix       = task.ext.prefix ?: "${meta.id}"
+    """
+    sentieon \\
+        driver \\
+        --algo GCBias --summary ${prefix}_gc_summary.txt ${prefix}_gc_metrics.txt \\
+        --algo MeanQualityByCycle ${prefix}_mq_metrics.txt \\
+        --algo QualDistribution ${prefix}_qd_metrics.txt \\
+        --algo InsertSizeMetricAlgo ${prefix}_is_metrics.txt  \\
+        --algo AlignmentStat ${prefix}_aln_metrics.txt \\
+        -t $task.cpus \\
+        $input \\
+        $ref \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        sentieon: \$(echo \$(sentieon driver --version 2>&1) | sed -e "s/sentieon-genomics-//g")
+    END_VERSIONS
+    """
+
+    stub:
+    def prefix       = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.mq_metrics.txt
```

Comment:
Here there is dot in file names instead of '_' as in the name of outputs in the script section. This works since it's * in the name of files in the output section but maybe it's better to keep the consistency.